src/main/java/dev/sigstore/fulcio/client/FulcioClient.java

```diff
@@ -122,26 +118,11 @@ public CertificateResponse SigningCert(CertificateRequest cr, String bearerToken
     if (sctHeader == null) {
```

Comment:
Fulcio doesn't have to send the SCT if the log isn't configured to send it - https://github.com/sigstore/fulcio/blob/main/pkg/api/legacy_server.go#L111-L119

Reply:
even if you can technically configure the fulcio server in no ct log mode `--ct-log-url=`

Reply:
It would be valid, if fulcio is self hosted for example. For the public insurance, there should always be an sct

Reply:
kk, I'll make it a config option on the client

Reply:
done